### ISSUE TITLE
Updates are currently disallowed on GET requests

### ISSUE THREAD

**Issue by karllarssonusawest** [SharePoint/sp-starter-kit#364]
#### Category
- [ ] Question
- [ x] Bug
- [ ] Enhancement

#### Expected or Desired Behavior

The 'Done' message, as per the 'Common SP Starter Kit Provisioning results' page: 

![image](https://user-images.githubusercontent.com/17866663/72025066-dcf13800-322b-11ea-8923-f2c3182c7126.png)

#### Observed Behavior

One of two errors:

1) I receive a 'task was cancelled' error after about 4 mins.

![image](https://user-images.githubusercontent.com/17866663/72025225-47a27380-322c-11ea-8b85-cb45b01862fb.png)

SharePoint lists the Starter Kit as deployed, but there are no new sites:

![image](https://user-images.githubusercontent.com/17866663/72025280-7c162f80-322c-11ea-89d9-f455ff9faf23.png)

2. I receive a "Updates are currently disallowed on GET requests.  To allow updates on a GET, set 
the 'AllowUnsafeUpdates' property on SPWeb" error after about 15 minutes:

![image](https://user-images.githubusercontent.com/17866663/72025385-cf887d80-322c-11ea-8ee8-a9fddb1cadd3.png)

With this error, the 3 sites are created and seem to function, but I have no way of telling if the process completed successfully.

Possibly relevant, but I also got that same 'AllowUnsafeUpdates' error when trying to create an App Catalog as part of the prerequisites. I switched to the classic admin experience and was able to create the App Catalog successfully there though.

In each case, when I try again, I delete the created sites (if applicable) and app from the App Store, but I have not yet figured out why I get different errors. It seems random.

I have also tried the Beta Provision Site, which also failed:

![image](https://user-images.githubusercontent.com/17866663/72025563-648b7680-322d-11ea-8417-d09e9e56efc0.png)

#### Steps to Reproduce

I am following the instructions on this page:

https://github.com/SharePoint/sp-starter-kit

...including the use of free developer tenant from the Office 365 developer program.

Attached is my full PowerShell output.

[debug.txt](https://github.com/SharePoint/sp-starter-kit/files/4037898/debug.txt)

**Comment by eoverfield** [comment]
@karllarssonusawest you are attempting to install the starter kit onto a recently created o365 developer tenant? If so, did you wait for at least 24 hours after you configured all of the pre-reqs before installing the kit?
The fact that the provisioning site service also failed you is definitely cause for a concern as I have found this service has been stellar at installing the kit on stubborn tenants.

**Comment by karllarssonusawest** [comment]
Thanks for the reply. I didn't wait the first time, but I deleted the package and did wait before trying again.

**Comment by eoverfield** [comment]
Were you then able to successfully install the kit? If so, please let us know as this will be helpful feedback for others I am sure. Thanks.

**Comment by karllarssonusawest** [comment]
No - I haven't had any success yet.

**Comment by karllarssonusawest** [comment]
I just tried again. In each case, I made sure there were no new sites and no deployed package before each run:

Attempt 1: Apply-PnPTenantTemplate : The object id "site:e63d12e5-6b53-4038-9bf5-b3fc0d182144:web:abebe78d-6934-4a9e-9607-fec6414269df" is invalid.

Attempt 2: Apply-PnPTenantTemplate : A task was canceled.

Attempt 3: Apply-PnPTenantTemplate : File Not Found.

Attempt 4: Updates are currently disallowed on GET requests.
